pep-0572.rst

```diff
@@ -898,7 +898,7 @@ The less confusing option is to make ``:=`` bind more tightly than comma.
 Always requiring parentheses
 ----------------------------
 
-It's been proposed to just always require parenthesize around an
+It's been proposed to just always require parentheses around an
```

Comment:
```suggestion
It has been proposed to always require parentheses around an
```

While we're at it:

* "It's" is short for "it is", not "it has"
* "just" is redundant here, and makes the sentence more complicated to parse.

Reply:
Good suggestions! Since this pull request just got merged, I'll make a separate pull request for these changes.

Reply:
I think we can skip these changes, since the existing text is suboptimal but not wrong. We should mostly avoid changes to finalized PEPs.

Reply:
The elimination of "just" was a little gratuitous, but the use of "It's" (which motivated the change) is an unambiguous error, as it is a contraction for "It is", not the grammatically correct (and presumably intended) "it has".

Reply:
"It's" can also be a contraction for "it has" (e.g., https://en.wiktionary.org/wiki/it%27s#Etymology_1, https://www.grammarly.com/blog/its-vs-its/#:~:text=It's%20is%20a%20contraction%20of,to%20or%20refers%20to%20something.).

Reply:
The current "It's" text is fine. It's just the parenthesis/parenthesize that is unambiguously wrong.